### PR TITLE
Extract Global Apollo Types from Domain | Closes #2448

### LINKED ISSUE #2448: Extract Global Apollo Types from Domain
Currently in `/domain/eventEditor/services/apollo/types.ts` we have some base types that are not specific to the EDTR at all and often referenced in code outside of `/domain/eventEditor/`. These include:

- EntityId
- EntityDbId
- Entity
- EntityEdge
- Trashable

these should all be moved into the application layer since they are not specific to the editor and will most certainly be required for other use cases (domains).

tasks:

- create new `types.ts` file in `/application/services/apollo/`
- move type definitions listed above into this new file
- update all paths and imports accordingly

### PR DESCRIPTION
**Pull request by manzoorwanijk**
This PR:
- Extracts global types from `/domain/eventEditor/services/apollo/types.ts` to `/application/services/apollo/types.ts`
- Updates the imports and paths
- Closes #2448

### CODE REVIEW

File: assets/src/application/ui/layout/entityActionsMenu/useEntityActionsMenuItems.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
 	AdditionalSubscriptionCbOptions,
 } from '@appLayout/entityActionsMenu';
 import { Domain } from './types';
-import { Entity } from '../../../../domain/eventEditor/services/apollo/types';
```
Comment (nicolad) [inline assets/src/application/ui/layout/entityActionsMenu/useEntityActionsMenuItems.tsx]: :+1:

File: assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/types.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import { Entity, EntityId, Price, Ticket } from '../../../services/apollo/types';
+import { Price, Ticket } from '../../../services/apollo/types';
```
Comment (nicolad) [inline assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/types.ts]: these entities won't be shared with other parts of the app?
this is why they live in `eventEditor` services, right?

Comment (manzoorwanijk) [reply assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/types.ts]: Yes, they are specific to EDTR.

File: assets/src/domain/shared/entities/prices/predicates/selectionPredicates/index.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { filter, find, includes, propEq } from 'ramda';
 
-import { EntityId, EntityDbId, Price } from '../../../../../eventEditor/services/apollo/types';
+import { EntityId, EntityDbId } from '@appServices/apollo/types';
+import { Price } from '../../../../../eventEditor/services/apollo/types';
```
Comment (nicolad) [inline assets/src/domain/shared/entities/prices/predicates/selectionPredicates/index.ts]: why not `from '@edtrServices/..'` ?

Comment (manzoorwanijk) [reply assets/src/domain/shared/entities/prices/predicates/selectionPredicates/index.ts]: Tests fail

**Pull request merged**